doc/source/apis.rst

```diff
@@ -193,7 +193,7 @@ Example::
 Activity lifecycle handling
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. module:: android.activity
+.. module:: android.activitylifecycle
```

Comment:
`ActivityLifecycleCallbacks` are implemented into `android.activity` module.

See:
https://github.com/kivy/python-for-android/blob/4d50d7d883bb6313f37c22402ecfaefa689bbbed/pythonforandroid/recipes/android/src/android/activity.py#L71

`android.activitylifecycle` module does not exists. Better to just avoid this reference.